src/models/MyPlugin.js

```diff
@@ -0,0 +1,34 @@
+import {
+  DefaultPlugin,
+  DefaultData,
+} from 'leto-modelizer-plugin-core';
+import MyPluginDrawer from 'src/draw/MyPluginDrawer';
+import MyPluginMetadata from 'src/metadata/MyPluginMetadata';
+import MyPluginParser from 'src/parser/MyPluginParser';
+import MyPluginRenderer from 'src/render/MyPluginRenderer';
+import { name, version } from 'package.json';
+
+/**
+ * My plugin.
```

Comment:
Example of a potential plugin model.
OR
Template of plugin model.

Reply:
done